.github/workflows/pr_build.yaml

```diff
@@ -0,0 +1,102 @@
+name: Build Binaries (Pull Request)
+
+on:
+  pull_request:
+
+env:
+  BUILD_TYPE: RelWithDebInfo
```

Comment:
I think we can delete this now, this is a CMake specific thing which no longer applies.